src/lib/wizard/components/Deploy.svelte

```diff
@@ -316,7 +321,9 @@
     <Message type="warn" message="Upgradable contracts are not yet fully supported. This action will only deploy the implementation contract without initializing. <br />We recommend using <u><a href='https://github.com/OpenZeppelin/openzeppelin-upgrades' target='_blank'>openzeppelin-upgrades</a></u> package instead." />
   {/if}
 
-  {#if inputs.length > 0}
+  {#if isCompiling}
```

Comment:
We should also disable the Deploy button below if `isCompiling` is true, otherwise users might still be able to deploy with previous compilation info.